README.md

```diff
@@ -2,8 +2,15 @@
 
 [![CircleCI](https://circleci.com/gh/pytorch/hub.svg?style=svg)](https://circleci.com/gh/pytorch/hub)
 
-## Logistics
+## Running Benchmarks Locally
+See [Benchmarks README.md](benchmark/README.md)
+
+## Nightly CI runs
```

Comment:
Ok - I was actually going to ask separately what you think about cleaning up the top level structure since I didn't understand what it was for and as a user of the benchmarks part it looks messy at the top level and not as clear as i'd like how to access the new benchmarks. 

That said, what is the history of this and what is the ongoing plan for the oss inference benchmarks?  Are those supposed to be separate from the train+eval benchmarks we're adding now, or should they be combined?

Ideally I'd like the top level README to be pretty clear about what's in the repo and how to find it.  If the intent is to keep 2 parts of the repo separate from each other, may be putting them under 2 subdirectories and having nothing else in the top level would make that clearer?

Reply:
They should be separate for now. 
Yea ideally we'll keep them in two subdirs but that can be a followup ;)